docs/formats/zson.md

```diff
@@ -259,36 +112,44 @@ may include additional decorators to successively refine the union type for unio
 that live inside other union types. This allows an already-decorated value to be
 further decorated with its union type and provides a means to distinguish
 a union value's precise member type when it is otherwise ambiguous as described in
-[Section 3.4.4](#344-union-value).
+[Section 3.4.4](#244-union-value).
 
-### 3.3 Primitive Values
+### 2.3 Primitive Values
 
-There are 23 types of primitive values with syntax defined as follows:
+The syntax for the primitive values defined by the Zed data model
```

Comment:
```suggestion
The syntax for the primitive values defined by the [Zed data model](zed-data-model.md#51-primitive-types)
```

Reply:
I've tried to follow a convention where the first reference in the doc has the relevent hyperlink.  We could change it so that every reference is linked but we should be consistent.  If we want to do this, I would do the typing in a subsequent PR.